src/Okta.Sdk.UnitTests/FeaturesClientShould.cs

```diff
@@ -70,15 +69,16 @@ public async Task GetDependents()
                 "[{\"Description\":null,\"Id\":null,\"name\":\"TestFeature\",\"Stage\":{\"State\":null,\"Value\":null},\"Status\":null,\"Type\":null}]";
             mockResponse.StatusCode = 200;
             _ = mockRequestExecutor
-                .GetAsync(dependentsEndpoint, Arg.Any<Dictionary<string, string>>(), Arg.Any<CancellationToken>())
+                .ExecuteRequestAsync(Arg.Any<HttpRequest>(), Arg.Any<CancellationToken>())
                 .Returns(Task.FromResult(mockResponse));
 
             var testClient = new TestableOktaClient(mockRequestExecutor);
 
             var retrievedDependents = await testClient.Features.ListFeatureDependents(featureId).ToListAsync();
             retrievedDependents.FirstOrDefault().Name.Should().Be("TestFeature");
             _ = await mockRequestExecutor.Received(1)
-                .GetAsync(dependentsEndpoint, Arg.Any<Dictionary<string, string>>(), Arg.Any<CancellationToken>());
+                .Received(1)
```

Comment:
It seems we have `.Received(1)` twice, do we want that?

Reply:
Good catch, I'll fix.